state/pruner.go

```diff
@@ -287,6 +310,7 @@ func (p *Pruner) pruneABCIResToRetainHeight(lastRetainHeight int64) int64 {
 	}
 	if numPruned > 0 {
 		p.logger.Info("Pruned ABCI responses", "heights", numPruned, "newRetainHeight", newRetainHeight)
+		p.metrics.ABCIResultsBaseHeight.Set(float64(newRetainHeight))
```

Comment:
Similar question to my other comment: is it possible for `numPruned` to be 0, but `newRetainHeight` to change?

Reply:
Same answer as above, I will mark this as resolved.